ui/vic-ui-h5c/vic/src/vic-app/src/app/services/vm-view.service.ts

```diff
@@ -83,7 +80,8 @@ export class VicVmViewService {
                 try {
                     for (let objectId in data.data) {
                         if (data.data.hasOwnProperty(objectId)) {
-                            vchsArray.push(new VirtualContainerHost(data.data[objectId]));
+                            vchsArray.push(
+                                new VirtualContainerHost(data.data[objectId]));
```

Comment:
Since I have a validation logic inside the class which could throw an error if the response from the server is not intact. So I think it could stay there. Let me know if you think otherwise.